base/docs/helpdb.jl

```diff
@@ -864,7 +864,8 @@ takebuf_array
 doc"""
     download(url,[localfile])
 
-Download a file from the given url, optionally renaming it to the given local file name. Note that this function relies on the availability of external tools such as `curl`, `wget` or `fetch` to download the file and is provided for convenience. For production use or situations in which more options are needed, please use a package that provides the desired functionality instead.
+Download a file from the given url, optionally renaming it to the given local file name. Note that this function relies on the availability of external tools such as `curl`, `wget` or `fetch` to download the file and is provided for convenience. For production use or situations in which more options are needed, please use a package that provides the desired functionality instead. Platform specific details: In Linux / OS X with wget or curl, passing silent=false will display their progress messages. In windows, download requires a full URL, complete with the http:// prefix.
+
```

Comment:
Break this up into multiple lines using a [92-character line break](http://julia.readthedocs.org/en/latest/manual/documentation/#documentation).